### PR TITLE
feat(navigable-tab): add change detection trigger

### PR DESCRIPTION
**Pull request by adisreyaj**
## Description
When tabs are dynamically added or removed change detection will now be automatically triggered.

### CODE REVIEW

File: projects/components/src/tabs/navigable/navigable-tab-group.component.ts [hunk 1]
```diff
@@ -49,6 +49,10 @@ import { NavigableTabComponent } from './navigable-tab.component';
       </nav>
       <div class="divider"></div>
     </div>
+
+    <!-- When tabs input changes, change detection will be triggered automatically -->
```
Comment (aaron-steinfeld) [inline projects/components/src/tabs/navigable/navigable-tab-group.component.ts:53]: The way we handle this elsewhere is just wrapping querylists with `queryListAndChanges` util. This should be used everywhere we use contentChildren

Comment (adisreyaj) [reply projects/components/src/tabs/navigable/navigable-tab-group.component.ts:60]: Fixed.

File: projects/components/src/tabs/navigable/navigable-tab-group.component.ts [hunk 2]
```diff
@@ -49,23 +56,28 @@ import { NavigableTabComponent } from './navigable-tab.component';
...
+    <ng-container *ngIf="tabs.changes | async"></ng-container>
```
Comment (anandtiwary) [inline projects/components/src/tabs/navigable/navigable-tab-group.component.ts:61]: can you merge these observables on L80?

Comment (adisreyaj) [reply projects/components/src/tabs/navigable/navigable-tab-group.component.ts:61]: Yup. This line will be removed.

**Review by arjunlalb — APPROVED** [review]

**Pull request merged**